samples/plugins/postgres_jdbc/connectionMetadata.xml

```diff
@@ -0,0 +1,8 @@
+<?xml version="1.0" encoding="utf-8"?>
+
+<connection-metadata>
+  <database enabled='true' label='Database'>
```

Comment:
The label is optional and this value does match the platform defaults, so I think lets remove it.